src/actions/NewSensorActions.js

```diff
@@ -28,11 +31,23 @@ const updateLoggingDelay = loggingDelay => ({
   payload: { loggingDelay },
 });
 
+const updateSensor = sensor => dispatch =>
```

Comment:
I'm gonna make some suggestions to use `await` over `then` because it's a personal preference. Maybe there is a concrete reason why not to use `await` that I can't see. Maybe you prefer `then`. I don't really mind if it's changed or not.

```
const updateSensor = sensor => async dispatch => {
  await dispatch(VaccineActions.startSetLogInterval(sensor));
  await dispatch(VaccineActions.startSensorDisableButton(sensor.macAddress));
};
```